Dockerfile

```diff
@@ -65,17 +65,126 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
            # Packages to install \
            libsasl2-dev freetds-bin build-essential sasl2-bin \
            libsasl2-2 libsasl2-dev libsasl2-modules \
-           default-libmysqlclient-dev apt-utils curl rsync netcat locales  \
+           dirmngr apt-utils curl rsync netcat locales  \
```

Comment:
I know that it isn't part of this PR, but can we make this list alphabetically and have one package per line? The one package per line play better with git:
```
apt-utils
build-essential
curl
default-libmysqlclient-dev
dirmngr
freetds-bin
freetds-dev
git
gosu
libffi-dev
libkrb5-dev
libpq-dev
libsasl2-2
libsasl2-dev
libsasl2-modules
libssl-dev
locales
netcat
nodejs
rsync
sasl2-bin
sudo
```

Reply:
Sure. I can do that :)